### PR TITLE
Add support for roc

### PR DESCRIPTION
**Pull request by zunda-arrow**
Checklist:

Idk if I did this right

Im not sure why my package is refusing to build

-   [ ] The package builds locally with `./piston build-pkg [package] [version]`
-   [ ] The package installs with `./piston ppman install [package]=[version]`
-   [ ] The package runs the test code with `./piston run [package] -l [version] packages/[package]/[version]/test.*`
-   [x] Package files are placed in the correct directory
-   [x] No old package versions are removed
-   [x] All source files are deleted in the `build.sh` script
-   [x] `metadata.json`'s `language` and `version` fields match the directory path
-   [x] Any extensions the language may use are set as aliases
-   [x] Any alternative names the language is referred to are set as aliases.

### CODE REVIEW

**Comment by zunda-arrow** [comment]
probably should change this to use the source code at a certain commit sha

File: packages/roc/2023.2.15/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+wget "https://github.com/roc-lang/roc/releases/download/nightly/roc_nightly-linux_x86_64-2023-02-15-b054977.tar.gz" -O "roc.tar.gz"
```
Comment (Enderchief) [inline packages/roc/2023.2.15/build.sh]: image

`wget` does not work. use curl instead

**Review by Enderchief — CHANGES_REQUESTED** [review]

File: packages/roc/2023.2.15/build.sh (hunk 1 above)
Comment (zunda-arrow) [reply packages/roc/2023.2.15/build.sh]: THANK YOU

**Comment by Enderchief** [comment]
I think this language won't work because piston cannot access the internet when running code and the test file relies on importing from github

**Comment by Enderchief** [comment]
image

**Comment by zunda-arrow** [comment]
Oh good to know. Ill stop wasting time getting it running lol :thumbies:

**Comment by Enderchief** [comment]
Actually I think we can get this working. `basic-cli` can be cloned locally and used 

[src](https://github.com/roc-lang/roc/issues/4655#issuecomment-1336215883)

**Comment by zunda-arrow** [comment]
I thought about this for a bit. I think forcing you to use `basic-cli` for your roc code would kinda ruin the point of the language, to the point that I don't think its worth putting the time into getting to run.

**Pull request closed without merging**